### PR TITLE
Add Erase Windows VSS support to compute disk

### PR DESCRIPTION
**Pull request by upodroid**
updated user facing examples to use debian 9 instead of debian 8.

If this PR is for Terraform, I acknowledge that I have:

- [X] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [X] [Generated Terraform](https://github.com/GoogleCloudPlatform/magic-modules#generating-downstream-tools), and ran `make test` and `make lint` to ensure it passes unit and linter tests.
- [ ] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/third_party/terraform/tests) (for handwritten resources or update tests).
- [X] [Ran](https://github.com/hashicorp/terraform-provider-google/blob/master/.github/CONTRIBUTING.md#tests) relevant acceptance tests (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [X] Read the [Release Notes Guide](https://github.com/GoogleCloudPlatform/magic-modules/blob/master/.ci/RELEASE_NOTES_GUIDE.md) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:enhancement
compute: added `erase_windows_vss_signature` field to `google_compute_disk` (beta)
```

### CODE REVIEW

**Comment by c2thorn** [comment]
@upodroid Hey! Someone else was going to take up adding the multiwriter option, and I never assigned the task to them, so that is my fault. However you have a few other fields added here, do you mind adding a test? Particularly for `eraseWindowsVssSignature`

**Comment by upodroid** [comment]
I added a new test, however I'm receiving 50X from Google. Internal Error.

**Comment by c2thorn** [comment]
@upodroid Yeah looks like the feature is still in alpha, and isn't enabled by default in projects. we can remove this feature from this PR so we can merge your other changes?

**Comment by upodroid** [comment]
It is done. I updated the PR Title and changelog to refer to this.

File: third_party/terraform/tests/resource_compute_disk_test.go.erb [hunk 1]
```diff
@@ -775,3 +801,38 @@ resource "google_compute_disk" "foobar" {
 `, policyName, diskName)
 }
 <% end -%>
+
+
+<% unless version == 'ga' -%>
+func testAccComputeDisk_multiWriterAndWindowsVSS(firstDiskName, projectName, snapshotName, diskName string) string {
```
Comment (c2thorn) [inline third_party/terraform/tests/resource_compute_disk_test.go.erb:807]:
```suggestion
func testAccComputeDisk_WindowsVSS(firstDiskName, projectName, snapshotName, diskName string) string {
```

File: third_party/terraform/tests/resource_compute_disk_test.go.erb [hunk 2]
```diff
@@ -457,6 +457,32 @@ func TestAccComputeDisk_resourcePolicies(t *testing.T) {
 }
 <% end -%>
 
+<% unless version == 'ga' -%>
+func TestAccComputeDisk_multiWriterAndVSSWindows(t *testing.T) {
```
Comment (c2thorn) [inline third_party/terraform/tests/resource_compute_disk_test.go.erb:461]:
```suggestion
func TestAccComputeDisk_VSSWindows(t *testing.T) {
```

File: third_party/terraform/tests/resource_compute_disk_test.go.erb [hunk 3]
```diff
@@ -457,6 +457,32 @@ func TestAccComputeDisk_resourcePolicies(t *testing.T) {
...
+	firstDiskName := fmt.Sprintf("tf-test-%s", randString(t, 10))
+	snapshotName := fmt.Sprintf("tf-test-%s", randString(t, 10))
+	projectName := getTestProjectFromEnv()
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccComputeDisk_multiWriterAndWindowsVSS(firstDiskName, projectName, snapshotName, diskName),
```
Comment (c2thorn) [inline third_party/terraform/tests/resource_compute_disk_test.go.erb:474]:
```suggestion
				Config: testAccComputeDisk_WindowsVSS(firstDiskName, projectName, snapshotName, diskName),
```

**Pull request merged**